docs/source/features.rst

```diff
@@ -6,14 +6,18 @@ JSON Conversion
 ===============
 
 pyelasticsearch converts transparently between Python datastructures and JSON.
-In request bodies, all the standard conversions are made: strings, numeric
-types, nulls, etc. In addition, we convert ``datetime`` and ``date`` instances
-to the format ES understands: ``2012-02-23T14:26:01``. ``date`` objects are
-taken to represent midnight on their day.
 
-A future release will convert more types, like datetimes, to Python in
-responses.
+* In request bodies, all the standard conversions are made: strings, numeric
+  types, nulls, etc.
+* We convert ``datetime`` and ``date`` instances to the format ES understands:
+  ``2012-02-23T14:26:01``. ``date`` objects are taken to represent midnight on
+  their day.
+* Python sets are converted to ES lists (which, after all, have no concept of
+  ordering).
```

Comment:
besides, ES has a concept of ordering, it maintains order of any list when retrieving the document